frontend/src/components/profile/SeeMatchHistoryBtn.tsx

```diff
@@ -0,0 +1,12 @@
+import { Link } from 'react-router-dom'
+import styles from './SeeMatchHistoryBtn.module.css'
+
+const SeeMatchHistory = () => {
```

Comment:
Should we name this as the filename ?
```suggestion
const SeeMatchHistoryBtn = () => {
```

Reply:
Yes !